tests/unit/processor/delete/test_delete.py

```diff
@@ -6,9 +6,9 @@
 class TestDelete(BaseProcessorTestCase):
```

Comment:
```suggestion
class TestDeleter(BaseProcessorTestCase):
```
the file module be `.deleter.test_deleter`